EIPS/eip-666.md

```diff
@@ -0,0 +1,52 @@
+---
+eip: 666
+title: The Number of the Beast
+description: Two horns.
+author: Beelzebub (@בַּע-זְבוּב)
+discussions-to: https://hell.com
+status: Undead
+type: Core
+deceased: 2022-04-01
+---
+
+## Abstract
+
+The Address of the Beast is
+* _0x000000000000000000000000000000000000029A_
+
+## Motivation
+
+We propose to assign a pre-defiled Ethereum address to The Beast.
+
+In the coming End Times
+> [... no man might buy or sell, save he that had the mark, or the name of the beast, or the number of his name.](https://www.kingjamesbibleonline.org/Revelation-13-17)
+
+This implies that this address will be needed for the continued operation of the Ethereum network in the End Times.
+
+## Specification
+
+> [Here is wisdom. Let him that hath understanding count the number of the beast: for it is the number of a man; and his number is Six hundred threescore and six.](https://www.kingjamesbibleonline.org/Revelation-13-8/)
+
+*666* is the Decimal Number of the Beast.  Therefore, the Hexadecimal Address of the Beast is
+* *0x000000000000000000000000000000000000029A*
+
+## Rationale
+
+John of Patmos Revealed the Number of the Beast in AD 95, which Number was Assigned since Genesis.
```

Comment:
How was this verified? Was the hash of the number already known? This seems to have some provability issues.